sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaRegistryClient.cs

```diff
@@ -72,7 +73,8 @@ public virtual async Task<Response<SchemaProperties>> RegisterSchemaAsync(string
             scope.Start();
             try
             {
-                var response = await RestClient.RegisterAsync(groupName, schemaName, serializationType, schemaContent, cancellationToken).ConfigureAwait(false);
+                var jsonStringSchemaContent = $"\"{JsonEncodedText.Encode(schemaContent, JavaScriptEncoder.UnsafeRelaxedJsonEscaping)}\"";
```

Comment:
would it be worth hoisting this into a `const` format mask:

```c#
private const string SchemaFormatMask = @"\{0}\";

// ...

var stuff = string.Format(SchemaFormatMask, JsonEncodedText.Encode(...));
```

Not sure, since it's such as simple format.

Reply:
Fixed! Also, the mask provided above caused a bug on the service. Instead of encoding `"` at the start and end of the string, it put `\` at the start and end. 💥